dspace/config/registries/schema-person-types.xml

```diff
@@ -0,0 +1,68 @@
+<dspace-dc-types>
+
+    <dspace-header>
+        <title>Person Types</title>
+    </dspace-header>
+
+    <dc-schema>
+        <name>person</name>
+        <namespace>https://schema.org/Person</namespace>
+    </dc-schema>
+
+
+    <!--NO EQUIVALENT FOUND
+        - person.identifier.orcid > NOT FOUND
+        - person.identifier.staffid > NOT FOUND (optional but very specific id: person.isicV4, person.naics, taxID, vatID, …)
+        - person.contributor.other > NOT FOUND
+    -->
+
+    <dc-type>
+        <schema>person</schema>
+        <element>givenName</element>
+        <scope_note>Given name. In the U.S., the first name of a Person. This can be used along with familyName instead
+            of the name property.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>person</schema>
+        <element>familyName</element>
+        <scope_note>Family name. In the U.S., the last name of an Person. This can be used along with givenName instead
+            of the name property.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>person</schema>
+        <element>telephone</element>
+        <scope_note>The telephone number.</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>person</schema>
+        <element>knowsLanguage</element>
+        <scope_note>Of a Person, and less typically of an Organization, to indicate a known language. We do not
+            distinguish skill levels or reading/writing/speaking/signing here. Use language codes from the IETF BCP 47
+            standard.
+        </scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>person</schema>
+        <element>email</element>
+        <scope_note>Email address.</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>person</schema>
+        <element>birthDate</element>
+        <scope_note>Date of birth.</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>person</schema>
+        <element>jobTitle</element>
+        <scope_note>The job title of the person (for example, Financial Manager).</scope_note>
+    </dc-type>
+
+</dspace-dc-types>
```

Comment:
@paulo-graca :  I'm a little worried about adding too many new schemas at once, and how to explain which schema to use in each scenario.  While we definitely want to model our new Entities schemas based on schema.org, the suggestion to use `Thing` seems like it would need more discussion, as it seems to result in some duplicative metadata fields, which could result in confusion for DSpace users.

For example, these fields all start to get confusing:
* `dc.identifier` vs `thing.identifier`
* `dc.title` (which is where we store the current name of a Community/Collection/Item) vs `thing.name`
* `dc.title.alternative` vs `thing.alternativeName`

So, I guess I'm wondering whether we need to make this step immediately or whether it's something we need to consider more. The initial goal of expanding to schema.org schemas was simply to support *new metadata fields* that aren't reasonably described via DC.  However, with the addition of `Thing` it seems like we'd be adding in some fields that overlap with DC fields, and that starts to cause confusion.

If it's useful, we could talk through this in more detail in next week's Entities WG meeting.  I'm not against your idea, I'm just worried it might cause more confusion than it's worth at this time.

Reply:
It begins to feel as though two points of view are solidifying:

- we are trying to move as many concepts as possible to schema.org names; or
- we need a few concepts that don't exist in DC, and we prefer to look for them first in schema.org.

If so, we should decide which one we will use.

(I incline to the latter.)

Reply:
@mwoodiupui : Good summary, and I agree with your analysis.  I also am inclined toward the second option initially (only using schema.org when we identify concepts that don't exist in DC). But, it's probably a point to clarify in our next Entities WG meeting. 

My (personal) opinion is that adding duplicative schemas/fields might get confusing, not only for users, but also in our Browse By, Search and Submission configuration.  As a basic example, if Names/Titles are stored in several fields depending on Item/Entity Type (dc.title, thing.name, etc), then we likely would need to update corresponding Browse/Search/Submission configurations to use multiple fields. Using multiple fields for similar information may also affect/complicate other metadata-related configurations/features (OAI-PMH crosswalks, batch metadata editing, etc).

In any case, I'd like to ensure we all can get on the same page here, so I'll add this as a topic for next week's Entities Mtg.

Reply:
I agree with both of you! Sorry for being the unfocused element :)! I was focused on Schema.org, and was trying to find out what we could extract on it to give us the answers and totally forgot that we've already a default schema that might already have the answer. I'm totally aligned with both and think that we first should look at DC and after Schema. Saying this, please ignore my Thing.* suggestions since we can use DC to accomplish the same result.